### PR TITLE
[Merged by Bors] - chore: tidy various files

### PR DESCRIPTION
**Pull request by Ruben-VandeVelde**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Analysis/SpecialFunctions/Gamma/Basic.lean [hunk 1]
```diff
@@ -553,8 +553,7 @@ lemma integral_rpow_mul_exp_neg_mul_Ioi {a r : ℝ} (ha : 0 < a) (hr : 0 < r) :
   convert integral_cpow_mul_exp_neg_mul_Ioi (by rwa [ofReal_re] : 0 < (a : ℂ).re) hr
   refine _root_.integral_ofReal.symm.trans <| setIntegral_congr measurableSet_Ioi (fun t ht ↦ ?_)
   norm_cast
-  rw [← ofReal_cpow (le_of_lt ht), RCLike.ofReal_mul]
-  rfl
+  simp_rw [← ofReal_cpow (le_of_lt ht), RCLike.ofReal_mul, coe_algebraMap]
```
Comment (b-mehta) [inline Mathlib/Analysis/SpecialFunctions/Gamma/Basic.lean:556]:
```suggestion
  simp_rw [← ofReal_cpow ht.le, RCLike.ofReal_mul, coe_algebraMap]
```
I guess this should work, and it's a bit tidier

File: Mathlib/Analysis/SpecialFunctions/Trigonometric/Bounds.lean [hunk 2]
```diff
@@ -88,25 +88,25 @@ lemma sin_le_mul (hx : -(π / 2) ≤ x) (hx₀ : x ≤ 0) : sin x ≤ 2 / π * x
 /-- Half of **Jordan's inequality** for absolute values. -/
 lemma mul_abs_le_abs_sin (hx : |x| ≤ π / 2) : 2 / π * |x| ≤ |sin x| := by
   wlog hx₀ : 0 ≤ x
-  case inr => simpa using this (by rwa [abs_neg]) <| neg_nonneg.2 <| le_of_not_le hx₀
+  · simpa using this (by rwa [abs_neg]) <| neg_nonneg.2 <| le_of_not_le hx₀
```
Comment (b-mehta) [inline Mathlib/Analysis/SpecialFunctions/Trigonometric/Bounds.lean:91]: What's the justification for this change? I was under the impression that named cases are preferred to anonymous ones?

File: Mathlib/Analysis/SpecialFunctions/Trigonometric/Bounds.lean [hunk 3]
```diff
@@ -88,25 +88,25 @@ lemma sin_le_mul (hx : -(π / 2) ≤ x) (hx₀ : x ≤ 0) : sin x ≤ 2 / π * x
...
   rw [abs_of_nonneg hx₀] at hx ⊢
   exact (mul_le_sin hx₀ hx).trans (le_abs_self _)
 
 lemma sin_sq_lt_sq (hx : x ≠ 0) : sin x ^ 2 < x ^ 2 := by
   wlog hx₀ : 0 < x
   case inr =>
     simpa using this (neg_ne_zero.2 hx) <| neg_pos_of_neg <| hx.lt_of_le <| le_of_not_lt hx₀
-  rcases le_or_lt x 1 with hxπ | hxπ
```
Comment (b-mehta) [inline Mathlib/Analysis/SpecialFunctions/Trigonometric/Bounds.lean:99]: What's the justification for this change? `rcases`, unlike `cases'`, is encouraged syntax.

File: Mathlib/MeasureTheory/Measure/SeparableMeasure.lean [hunk 4]
```diff
@@ -220,14 +217,14 @@ theorem Measure.MeasureDense.of_generateFrom_isSetAlgebra_finite [IsFiniteMeasur
...
-                  exact (measure_mono <| iUnion_subset <|
-                    fun i ↦ iUnion_subset (fun _ ↦ subset_iUnion f i))
-                  exact iUnion_subset <| fun i ↦ iUnion_subset (fun _ ↦ subset_iUnion f i)
-                  exact MeasurableSet.biUnion (countable_coe_iff.1 inferInstance)
-                    (fun n _ ↦ (hf n).1.nullMeasurableSet)
+                  · apply lt_of_le_of_lt (sub_le_dist ..)
+                    simp only [Finset.mem_range, Nat.lt_add_one_iff]
+                    exact (dist_comm (α := ℝ) .. ▸ hN N (le_refl N))
+                  · exact (measure_mono <| iUnion_subset <|
+                      fun i ↦ iUnion_subset (fun _ ↦ subset_iUnion f i))
```
Comment (b-mehta) [inline Mathlib/MeasureTheory/Measure/SeparableMeasure.lean:224]:
```suggestion
                  · exact measure_mono <| iUnion_subset <|
                      fun i ↦ iUnion_subset fun _ ↦ subset_iUnion f i
```
looks like it should work?

File: Mathlib/Topology/FiberPartition.lean [hunk 5]
```diff
@@ -53,7 +53,7 @@ def sigmaInclIncl {X : Type*} (g : Y → X) (a : Fiber (g ∘ f))
     have := x.prop
     simp only [sigmaIncl, ContinuousMap.coe_mk, Fiber.mem_iff_eq_image, comp_apply] at this
     rw [Fiber.mem_iff_eq_image, Fiber.mk_image, this, ← Fiber.map_preimage_eq_image]
-    rfl⟩
+    simp [sigmaIncl]⟩
```
Comment (b-mehta) [inline Mathlib/Topology/FiberPartition.lean:56]: What's going on here? Was this a heavy-rfl before?

File: Mathlib/Analysis/SpecialFunctions/Trigonometric/Bounds.lean (hunk 2 above)
Comment (Ruben-VandeVelde) [reply Mathlib/Analysis/SpecialFunctions/Trigonometric/Bounds.lean:91]: I'm not sure that's a rule, and in any case I don't think the `inr` name from `wlog` is intentional or even a good name.

File: Mathlib/Topology/FiberPartition.lean (hunk 5 above)
Comment (Ruben-VandeVelde) [reply Mathlib/Topology/FiberPartition.lean:56]: Not so heavy, but seeing through several definitions, so this should be more robust.

**Review by Ruben-VandeVelde — COMMENTED** [review]
Thanks!

File: Mathlib/Analysis/SpecialFunctions/Trigonometric/Bounds.lean (hunk 2 above)
Comment (b-mehta) [reply Mathlib/Analysis/SpecialFunctions/Trigonometric/Bounds.lean:91]: I don't like it but that's a fair point :)

File: Mathlib/Analysis/SpecialFunctions/Trigonometric/Bounds.lean (hunk 3 above)
Comment (Ruben-VandeVelde) [reply Mathlib/Analysis/SpecialFunctions/Trigonometric/Bounds.lean:99]: I'm not sure if the case names from `rcases` are stable; in any case there's no advantage to using `rcases` here, and separating the case names from the names of the introduced hypotheses is less readable, IMO.

Comment (b-mehta) [reply Mathlib/Analysis/SpecialFunctions/Trigonometric/Bounds.lean:99]: The case names from `rcases` correspond exactly with the case names from `cases`, so this change doesn't affect stability in that sense. There's also no advantage to using `cases` here, the introduced hypotheses aren't referred to by name in the proof.

And in later cases, the `rfl` syntax for `rcases` makes the proof clearer and simpler to read, so there is a stronger advantage which is not allowed with the `cases` proof.

Comment (Ruben-VandeVelde) [reply Mathlib/Analysis/SpecialFunctions/Trigonometric/Bounds.lean]: I've dropped the case-related changes from this PR

File: Mathlib/MeasureTheory/Function/LocallyIntegrable.lean [hunk 6]
```diff
@@ -127,8 +127,8 @@ theorem LocallyIntegrableOn.aestronglyMeasurable [SecondCountableTopology X]
   rw [this, aestronglyMeasurable_iUnion_iff]
   exact fun i : ℕ => (hu i).aestronglyMeasurable
 
-/-- If `s` is either open, or closed, then `f` is locally integrable on `s` iff it is integrable on
-every compact subset contained in `s`. -/
+/-- If `s` is locally closed (e.g. open or closed), then `f` is locally integrable on `s` iff it is
+integrable on every compact subset contained in `s`. -/
 theorem locallyIntegrableOn_iff [LocallyCompactSpace X] (hs : IsLocallyClosed s) :
     LocallyIntegrableOn f s μ ↔ ∀ (k : Set X), k ⊆ s → (IsCompact k → IntegrableOn f k μ) := by
```
Comment (b-mehta) [inline Mathlib/MeasureTheory/Function/LocallyIntegrable.lean:133]:
```suggestion
    LocallyIntegrableOn f s μ ↔ ∀ (k : Set X), k ⊆ s → IsCompact k → IntegrableOn f k μ := by
```

Comment (b-mehta) [reply Mathlib/MeasureTheory/Function/LocallyIntegrable.lean:133]: optional but seems in spirit with this PR!

**Review by b-mehta — APPROVED** [review]
bors d+

**Pull request closed without merging**